mp/src/materialsystem/stdshaders/pbr_ps30.fxc

```diff
@@ -333,5 +359,11 @@ float4 main( PS_INPUT i ) : COLOR
 	//return float4(g_LightmapCoords[0].x + i.baseTexCoord.x * g_LightmapCoords[0].z, g_LightmapCoords[0].z, i.baseTexCoord.x, 1);
 	//return tex2D(LightmapSampler, g_LightmapCoords[0].xy + i.baseTexCoord * g_LightmapCoords[0].z);
 
-	return FinalOutput( float4( directLighting + ambientLighting, alpha ), fogFactor, PIXELFOGTYPE, TONEMAP_SCALE_LINEAR, bWriteDepthToAlpha, i.projPos.z );	
+#if EMISSION
+	float3 combinedLighting = (directLighting + ambientLighting) * ambientOcclusion + emission;
```

Comment:
Would you mind setting `combinedLighting` above the `#if` and doing `combinedLighting += emission;` inside the `#if`?

Reply:
alright